docs/sink-configuration.md

```diff
@@ -75,3 +75,9 @@ If `HAWKULAR_SERVER_URL` includes any path, the default `hawkular/metrics` is ov
 The following options are available:
 
 * `tenant` - Hawkular-Metrics tenantId (default: `heapster`)
+
+## Modifying the sinks at runtime
```

Comment:
Note that `--insecure` is necessary for curl to work with the self-signed SSL certificate.